### PR TITLE
Release/2.3.0

### PR DESCRIPTION
**Pull request by krystofwoldrich**
Since changelogs crashed the merge step of release I assume we need to merge this manually?

The `2.3.0` was successfully released to npm and GitHub, just the merge failed.

https://github.com/getsentry/publish/issues/1502
https://www.npmjs.com/package/@sentry/wizard/v/2.3.0

#skip-changelog

### CODE REVIEW

**Review by vladanpaunovic — APPROVED** [review]
LGTM

**Comment by marandaneto** [comment]
Wouldn't be better to just merge the branch `v2.3.0` into the main branch and solve the conflicts?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 # Changelog
 
 ## Unreleased
-- chore(ci): remove jira workflow (#204)
+
+
+
```
Comment (marandaneto) [inline CHANGELOG.md:6]:
```suggestion

```

**Comment by marandaneto** [comment]
> Wouldn't be better to just merge the branch `v2.3.0` into the main branch and solve the conflicts?

Oh I guess that's what you are doing, but from the release branch and not the tag.

**Review by vladanpaunovic — APPROVED** [review]

**Pull request merged**